.circleci/config/commands/check-branch-name.yml

```diff
@@ -0,0 +1,14 @@
+description: >
+  Check if branch name starts with ui/ or docs/ and if so, exit.
+steps:
+  - run:
+      working_directory: ~/
+      name: Check branch name
+      command: |
+        # If the branch being tested starts with ui/ or docs/ we want to exit stop the job without failing
+        [[ "$CIRCLE_BRANCH" = ui/* || "$CIRCLE_BRANCH" = docs/* ]] && {
+          # stop the job from this step
+          circleci-agent step halt
+        }
+        # exit with success either way
+        exit 0
```

Comment:
Exiting `0` is default behaviour, so I might leave this line out.

Reply:
It seemed to exit 1 when I didn't include it... but maybe that was for a different reason. I'll test without it